README.md

```diff
@@ -7,66 +7,47 @@
 
```

Comment:
- [ ] Please ensure you add an appropriate Project title and description.

  ![ishpaul-title](https://user-images.githubusercontent.com/54848363/177315068-d31ea638-263e-4bb8-aa8e-c24948ad3087.jpg)